packages/templates/test/utils/createUtil.test.ts

```
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { nls } from '../../src/i18n';
+import { CreateUtil } from '../../src/utils';
+
+import { assert, expect } from 'chai';
+import * as fs from 'fs';
+import * as path from 'path';
+import { SinonStub, stub } from 'sinon';
+
+/* tslint:disable: no-unused-expression */
+describe('CreateUtil', () => {
+  describe('checkInputs', () => {
+    it('should throw error for input with non-alphanumeric characters', () => {
```

Do we need to check for empty inputs ?

---

yeah it should throw an AlphaNumericNameError error too. I'll add that test case.